onnx/onnx.in.proto

```diff
@@ -451,6 +451,9 @@ message GraphProto {
   // Initializers (see above) stored in sparse format.
   repeated SparseTensorProto sparse_initializer = 15;
 
+  // Initializers (see above) stored in sparse format.
+  repeated SequenceProto sequence_initializer = 16;
```

Comment:
Should we add something for "maps" and "optional" types at the same time? There is no request for those currently, but it helps to avoid multiple bumps to IR_version if we do them all together. What do you think? Thanks!